packages/styles/layouts/_l.flexy.scss

```diff
@@ -9,7 +9,7 @@
     }
   }
 
-  &--space-aroud {
+  &--space-around {
```

Comment:
I you change that you need to change the previews that depends on this class. 
the previews are html and css files that are used in the docs to demonstrate the classes
into `/Foundations/Layout/Grid/previews`

Reply:
I think there is only one using it

Reply:
Done